### ISSUE TITLE
[PGS support]PGS diskful deployment support without manual step

### ISSUE THREAD

**Issue by zet809** [xcat2/xcat-core#2935]
1， Trace and work around udevadmin issue ：https://bugzilla.linux.ibm.com/show_bug.cgi?id=153609
2， Trace issue #2905

**Comment by bybai** [comment]
Discussed in the meeting, closed this task, and open a new task https://github.com/xcat2/xcat-core/issues/3011 to trace LTC issue.

**Comment by bybai** [comment]
I verified， do not need "rd.dm=0 rd.md=0 nodmraid" in kcmdline, default petitboot file is enough for diskful installation. For example:
```
]# vi p9euh01
#install rhels7.4-pegas-by-ppc64le-compute
default xCAT
label xCAT
        kernel http://10.6.29.1:80/tftpboot/xcat/osimage/rhels7.4-pegas-by-ppc64le-install-compute/vmlinuz
        initrd http://10.6.29.1:80/tftpboot/xcat/osimage/rhels7.4-pegas-by-ppc64le-install-compute/initrd.img
        append "quiet inst.repo=http://10.6.29.1:80/install/rhels7.4-pegas-by/ppc64le inst.ks=http://10.6.29.1:80/install/autoinst/p9euh01 BOOTIF=70:e2:84:14:24:84  ip=dhcp  inst.loglevel=debug inst.syslog=10.6.29.1 inst.sshd"
```

**Comment by bybai** [comment]
Fixed in #3038
